sim/dex.js

```diff
@@ -186,6 +186,9 @@ class ModdedDex {
 		return this.formatsCache;
 	}
 	/**
+	 * A map from mod names to moddedDex instances.
+	 * This getter will create the necessary ModdedDex instances if
+	 * 	they do not yet exist.
```

Comment:
This is weird indentation.

Also, it's actually a map from mod IDs to Dex instances. It should also possibly not be documented since it's not meant to be read directly; it should be accessed with `Dex.mod`.